src/ddmd/blockexit.d

```diff
@@ -94,7 +98,7 @@ int blockExit(Statement s, FuncDeclaration func, bool mustNotThrow)
 
         override void visit(ErrorStatement s)
         {
-            result = BEany;
+            result = BE.none;
```

Comment:
That doesn't look like a refactor...

Reply:
This is completely trivial. My complaint with mixing refactoring with semantic changes is when there'd be a hundred lines of code moved about that completely obfuscated what the actual fix was. That is not the case here.